pallets/loans/src/tests.rs

```diff
@@ -556,6 +572,20 @@ fn ratio_and_rate_works() {
     })
 }
 
+#[test]
```

Comment:
Need to put related tests together.

Reply:
I mean tests cases near to each other, but still seperate each case with a standalone function.